l10n-usa-account-profile/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "US Accounting",
+    "summary": "Additional features to manage your accounting in Odoo",
```

Comment:
I think it would be good to include US in the summary too - the UI shows the summary in place of the name on hover IIRC, so the fact that this is US would become obscured and require a toggle back. Maybe this?

> Additional features to manage USA accounting in Odoo